src/modules/twap/containers/TwapConfirmModal/TwapConfirmDetails.tsx

```diff
@@ -1,16 +1,19 @@
+import React from 'react'
+
 import { ConfirmDetailsItem } from 'modules/trade/pure/ConfirmDetailsItem'
 import { ReviewOrderModalAmountRow } from 'modules/trade/pure/ReviewOrderModalAmountRow'
 
 import { PartsState } from '../../state/partsStateAtom'
 
 export type TwapConfirmDetailsProps = {
   startTime: number | undefined
-  partDuration: number | undefined
+  partDuration: string
+  totalDuration: string
```

Comment:
are we sure the formatting is not part of the responsibility of this component? 
I would think, `partDuration: number`  + `numberOfParts: number` should be enough params